src/sass/app.scss

```diff
@@ -1,3 +1,10 @@
-.example-global {
-	color: red;
-}
\ No newline at end of file
+// Global styles go here...
```

Comment:
Probably just let these be global for now. It's not clear to me if we'll actually need them in the component - and if we do, we can put them in the app vue file.

Reply:
Left this global for now.

Reply:
These aren't actually global anymore...

Reply:
I don't know if these styles / this file is necessary?